### DISCUSSION TITLE
Ability to add lib32, base at install

### DISCUSSION THREAD

**Discussion by sdo992** [helloSystem/hello#397] [General]
First, thank you for this hard work. So far, I love the aesthetics and I agree with the philosophy. However, it's still a FreeBSD base, which means nothing is easy. For example, my ultimate goal is to install wine so I can run the couple of programs I still have that use Windows.

I've tried to reinstall helloSystem 3 or 4 times now without any luck. I just borked the last install trying to download the base and lib32 from:

http://ftp.freebsd.org/pub/FreeBSD/releases/amd64/13.0-RELEASE/

And then trying:

```
sudo tar -C / -xvpf lib32.txz
sudo tar -C / -xvpf base.txz
```

The problem was that the base.txz overwrote the /etc/master.passwd file and I was locked out of the system. So, I came back to the installer, just now, and I'm going to try to install from bsdinstaller. I am hoping that it install the entire DE along with it, but I'm not hopeful.

I could not find any simple directions to install lib32, at all.

EDIT: The bsdinstall just failed with a message about not being able to find the MANIFEST.

**Comment by probonopd** [comment]
> The problem was that the base.txz overwrote the /etc/master.passwd file and I was locked out of the system. 

Hello @sdo992. `base.txz` contains the FreeBSD operating system. The way I understand it you are never ever supposed to extract it over an already set up, running system (like helloSystem). You can do that with `lib32.txz` and with `source.txz` but __not__ with `base.txz`.

> So, I came back to the installer, just now, and I'm going to try to install from bsdinstaller. 

To install helloSystem, run the Live ISO, and use "Install FreeBSD" from the Menu.
